core/system/src/main.c

```diff
@@ -50,6 +51,11 @@ UVISOR_NOINLINE void uvisor_init_post(void)
         /* init MPU */
         vmpu_init_post();
 
+        /* init page allocator */
```

Comment:
Please for the moment make sure that your code adheres to the style guide, it helps the team concentrating on the real stuff (it's 20 commits) and it keeps the techdebt down. I love the idea of the script, but I think it won't be available by the time we need this PR merged.

Also, this applies to this PR only, next ones will have their style right!